tests/serializable.cpp

```
@@ -0,0 +1,95 @@
+#include "utility/serializable.hpp"
+
+#include <yaml-cpp/yaml.h>
+
+#include <vector>
+#include <gtest/gtest.h>
+
+using namespace rmcs::util;
```

Clang (14.0.6)

[warning] do not use namespace using-directives; use using-declarations instead

(google-build-using-namespace)